bundles/org.openhab.binding.ecoflow/README.md

```diff
@@ -0,0 +1,158 @@
+| battery-input#temperature     | Number:Temperature       | Battery temperature                          | Yes       |         |
+| battery-input#active          | Switch                   | Whether battery input is active              | Yes       |         |
+| battery-input#battery-level   | Number                   | Charge level (0-100%) of connected battery   | Yes       |         |
+| battery-input#charge-limit    | Dimmer                   | Upper battery charge limit (0-100%)          | No        |         |
+| battery-input#discharge-limit | Dimmer                   | Lower battery discharge limit (0-100%)       | No        |         |
+| pv1-input#voltage             | Number:ElectricPotential | Panel 1 voltage                              | Yes       |         |
+| pv1-input#voltage-target      | Number:ElectricPotential | Panel 1 MPPT target voltage                  | Yes       |         |
+| pv1-input#current             | Number:ElectricCurrent   | Panel 1 current                              | Yes       |         |
+| pv1-input#power               | Number:Power             | Panel 1 power                                | Yes       |         |
+| pv1-input#mppt-active         | Switch                   | Whether panel 1 MPPT is active               | Yes       |         |
+| pv2-input#voltage             | Number:ElectricPotential | Panel 2 voltage                              | Yes       |         |
+| pv2-input#voltage-target      | Number:ElectricPotential | Panel 2 MPPT target voltage                  | Yes       |         |
+| pv2-input#current             | Number:ElectricCurrent   | Panel 2 current                              | Yes       |         |
+| pv2-input#power               | Number:Power             | Panel 2 power                                | Yes       |         |
+| pv2-input#mppt-active         | Switch                   | Whether panel 2 MPPT is active               | Yes       |         |
+
+Remarks:
+
+- [1] Possible states: 'idle', 'starting', 'synchronized' and 'disconnected'.
+- [2] The range of valid values is 0..600/800 W. The upper limit depends on inverter type.
+- [3] Possible states: 'prioIsSupply' (prefer AC output over battery storage), 'prioIsStorage' (prefer battery storage over AC output)
+
+## File Based Configuration
+
+If you want to create the API bridge in a .things file, the entry has to look as follows:
+
+```java
+Bridge ecoflow:ecoflowapi:ecoflowapi [ accessKey="YOUR_ACCESS_KEY", secretKey="YOUR_SECRET_KEY" ]
+```
+
+The devices are detected automatically.
+If you also want to enter those manually, the syntax is as follows:
+
+```java
+Bridge ecoflow:ecoflowapi:ecoflowapi [ accessKey="YOUR_ACCESS_KEY", secretKey="YOUR_SECRET_KEY" ]
+{
+    Thing delta2max myPowerStation "EcoFlow power station" [ serialNumber="serial number as printed on device" ]
+}
+```
+
```

Comment:
Coudl you also add `ecoflow.items` example?

Reply:
Understand your doubt. In the past it had more use, as of recent builds there are functions added to generate DSL code into main UI. But for now it is still a required step in the readme.md and for some it is still usefull. You might create an issue to discuss if this tempalte should be updated or not.